packages/graphiql/package.json

```diff
@@ -101,7 +101,7 @@
     "webpack": "5.76.0",
     "webpack-bundle-analyzer": "^3.6.1",
     "webpack-cli": "^5.0.1",
-    "webpack-dev-server": "^4.11.1",
+    "webpack-dev-server": "^4.13.1",
```

Comment:
Can we leave that version unchanged? Otherwise we'd be installing two versions for `webpack-dev-server` right now. Seems to me that the fix also works with `4.11.1`.